### ISSUE TITLE
Unable to import musicxml exported from MuseScore 4

### ISSUE THREAD

**Issue by milnak** [jjazzboss/JJazzLab#461]
# Repro

In [MuseScore 4](https://musescore.org/en) open a chart with chords.

File/Export:

* Main score
* Format: Music XML
* File type: Uncompressed (*.musicxml)

Click "export..."

Rename the exported file to "name.xml"

**Issue 1:** JJazzLab doesn't support "musicxml" extension.

Open jjazzlab and choose File > Import Songs...

open "name.xml".

**Issue 2:** Receive the following:

```text
nu.xom.ParsingException: The markup declarations contained or pointed to by the document type
declaration must be well-formed. at line 1, column 2 in http://www.musicxml.org/dtds/partwise.dtd
```

Sample file: [example.zip](https://github.com/user-attachments/files/16554638/example.zip)

**Comment by jjazzboss** [comment]
Actually this is not a bug, it's just that the referenced dtd `http://www.musicxml.org/dtds/partwise.dtd` in your file is not available anymore.  I found other people complaining about this on MuseScore forum.

I found that W3C has put the dtd files on GitHub, so I replaced the original dtd by `https://raw.githubusercontent.com/w3c/musicxml/v4.0/schema/partwise.dtd` then JJazzLab successfully imported your file.
